test/pytest.ini

```diff
@@ -23,4 +23,5 @@ env =
     OBJECT_STORAGE__BUCKET_NAME=test-object-storage
     OBJECT_STORAGE__PRESIGNED_URL_EXPIRY_SECONDS=1800
     ATTACHMENT__MAX_SIZE_BYTES=100
+    ATTACHMENT__UPLOAD_LIMIT=50
```

Comment:
Should this be lowered and have an e2e test?

Reply:
I don't think there are any cases we need more than one at the moment? So could probably even do 2 but I doubt it should matter much - if we need to raise it for others tests in future we can.